Ascend.java

```diff
@@ -1,7 +1,17 @@
+import javax.imageio.ImageTranscoder;
+
 // Generates three integer random numbers in a given range,
 // and prints them in increasing order.
 public class Ascend {
 	public static void main(String[] args) {
-		// Replace this comment with your code
+		int lim = Integer.parseInt(args[0]);
+		int a = (int) (Math.random()*lim);
+		int b = (int) (Math.random()*lim);
+		int c = (int) (Math.random()*lim);
+		int maxVal = Math.max(a,Math.max(b,c));
+		int minVal = Math.min(a,Math.min(b,c));
+		int midVal= (a+b+c)- maxVal-minVal;
+		System.out.println((int) a+" "+(int) b+" "+(int) c);

```

Comment:
The explicit cast to int in the System.out.println statement is unnecessary because the variables a, b, and c are already declared as integers. 
Casting them again to int does nothing